keyboards/qwertykeys/qk65/solder/rules.mk

```diff
@@ -15,4 +15,6 @@ COMMAND_ENABLE = no         # Commands for debug and configuration
 NKRO_ENABLE = yes           # Enable N-Key Rollover
 BACKLIGHT_ENABLE = no       # Enable keyboard backlight functionality
 RGBLIGHT_ENABLE = no        # Enable keyboard RGB underglow
-AUDIO_ENABLE = no           # Audio output
\ No newline at end of file
+AUDIO_ENABLE = no           # Audio output
+
+LAYOUTS = 65_ansi_blocker 65_ansi_blocker_split_bs 65_ansi_blocker_tsangan 65_ansi_blocker_tsangan_split_bs 65_iso_blocker 65_iso_blocker_split_bs
```

Comment:
```suggestion
LAYOUTS = 65_ansi_blocker 65_ansi_blocker_split_bs 65_ansi_blocker_tsangan 65_iso_blocker 65_iso_blocker_split_bs
```

Reply:
I forgot I haven't posted that branch to PR yet. :eyes: